src/styles/fonts/_fontawesome.scss

```diff
@@ -0,0 +1,2199 @@
+/*!
```

Comment:
I will suggest to use svg images for icon stuff.

But that is true that technology stack is confusing e.g. i spent time to figure out that  ```.navbar``` is part of boostrap and we are using bootstrapUI.

So i thought to to add technology stack in "contributing.md" for future reference

e.g.

HTML - Plain HTML + Reactjs Component
CSS - SCSS + Bootstrap
Client-Side JS - Reactjs+Redux
Server-side JS - Express 

right?

then i have a question, not sure to ask that here.

which browser stack we are supporting? because usage of HTML/CSS/JS is really dependent on that stack, and will help developers and code-reviewers to evaluate a solution. Thanks